Frontend/ui-driver/src/Screens/BookingOptionsScreen/Controller.purs

```diff
@@ -162,6 +162,11 @@ getVehicleCapacity vehicleType = case vehicleType of
   "TAXI" -> getString ECONOMICAL <> " · " <> "4 " <> getString PEOPLE
   "SUV" -> getString SPACIOUS <> " · " <> "6 " <> getString PEOPLE
   "BIKE" -> getString ECONOMICAL <> " · " <> "1 " <> getString PEOPLE -- todo-codex: Add proper string for vehicle capacity
+  "AMBULANCE_TAXI" -> "Ambulance Taxi" <> " · " <> "4 " <> getString PEOPLE
```

Comment:
Translations for every string, I would suggest use a string like `Economical` or `Budget Friendly`.